src/rules.js

```diff
@@ -1841,6 +1841,39 @@ module.exports = [
     },
     /**
      ==================================================================
+     TEXT-INDENT
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        name: 'Text indent',
+        prefix: 'Ti-',
+        properties: ['text-indent'],
+        allowCustom: true,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'inh', values: ['inherit']}
+        ]
+    },
+    /**
+     ==================================================================
+     TEXT-OVERFLOW
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        name: 'Text overflow',
+        prefix: 'Tov-',
+        properties: ['text-overflow'],
+        allowCustom: true,
+        allowSuffixToValue: false,
```

Comment:
People should be able to do that using a custom value, but I preferred to set `allowSuffixToValue` to `false` to prevent potential parsing issues.